kubehelpers/configMap.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 gRPC authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubehelpers
+
+import (
+	"encoding/json"
+)
+
+// UpdateConfigMapWithServerPort accepts a server port string and a scenarioString string.
+// It returns a updated scenarioString with server port inserted. Currently only supports
+// single scenario.
+// TODO: wanlin31 to update the function to support array of scenarios.
```

Comment:
Format should be TODO(wanlin31): update the function to support an array of scenarios.